src/parser.js

```diff
@@ -1,22 +1,34 @@
-const {
-  isDocumentTypeNode,
-  isTextNode,
-  isElementNode,
-  isCommentNode
-} = require('parse5').treeAdapters.default
+const DOCUMENT_TYPE_NODE = '#documentType'
+const TEXT_NODE = '#text'
+const DIV_NODE = 'div'
+const COMMENT_NODE = '#comment'
+const COMMENT_NODE_PUG = '//'
+
+const hasSingleTextNodeChild = node => {
+  return (
+    node.childNodes &&
+    node.childNodes.length === 1 &&
+    node.childNodes[0].nodeName === TEXT_NODE
+  )
+}
 
 class Parser {
-  constructor ({ root, tabs = false }) {
+  constructor({ root, useTabs = false }) {
     this.root = root
-    this.tabs = tabs
+    this.useTabs = useTabs
     this.pug = ''
   }
 
-  get indent () {
-    return this.tabs ? '\t' : '  '
+  // Tab vs. Space
+  get indentType() {
```

Comment:
Why not setting this in the constructor?

Reply:
I blindly changed a legacy method without realising that it would fit better in the constructor. Nice spot.